r/README.md

```diff
@@ -143,7 +143,15 @@ checkout:
 
 ``` shell
 cd ../../r
-R -e 'install.packages(c("devtools", "roxygen2", "pkgdown", "covr")); devtools::install_dev_deps()'
+
+R --vanilla << EOF
```

Comment:
These instructions are intended for Windows users too (at least nominally) and this `<< EOF` heredoc trick does not work on Windows.

Reply:
I think it's also confusing; I'd go back to the one-liner, that wasn't the problem.

Reply:
I changed this to Rscript instead, thanks !!